core/src/main/java/org/mule/runtime/core/api/config/QueueProfile.java

```diff
@@ -55,8 +55,7 @@ public void setMaxOutstandingMessages(int maxOutstandingMessages) {
     this.maxOutstandingMessages = maxOutstandingMessages;
   }
 
-  public QueueConfiguration configureQueue(String component, QueueManager queueManager)
-      throws InitialisationException {
+  public QueueConfiguration configureQueue(String component, QueueManager queueManager) throws InitialisationException {
```

Comment:
We would be breaking compilation wherever this method is called and the `InitialisationException` is not handled. However it would break at compile time instead of runtime, so maybe we could go forward with it.

Reply:
Actually it would break binary compatibility.

Reply:
:piccard-facepalm: